### PR TITLE
Added functionality to dynamically add columns

### PR DESCRIPTION
**Pull request by probro27**
## Summary of Changes
Added a helper function `addSQLColumnIfNotExists` in `db.ts` to add a column if it does not exist in the database table. 

## Motivation and Explanation
This will allow developers to use this function and add columns without having to write the entire logic. 

## Related Issues
#418 

## Demonstration of Changes
The profile emoji functionality seems to be working fine.

### CODE REVIEW

File: src/components/db.ts [hunk 1]
```diff
@@ -148,6 +140,23 @@ const initRpsGameInfo = async (db: Database): Promise<void> => {
   );
 };
 
+const addSQLColumnIfNotExists = async (
```
Comment (mcpenguin) [inline src/components/db.ts:143]: Can we add some documentation for this function? Explain what its purpose is, as well as what the different parameters are. We should specify that `columnDataType` is a SQL data type, and give an example (say "eg `VARCHAR(32)`").

**Review by mcpenguin — CHANGES_REQUESTED** [review]
Looks good, just had a small change request.

File: src/components/db.ts (hunk 1 above)
Comment (probro27) [reply src/components/db.ts:143]: Got it. I'll add a small documentation comment for this.

**Review by mcpenguin — APPROVED** [review]
LGTM!

**Pull request merged**